RELEASING

```diff
@@ -0,0 +1,25 @@
+# Scala.js Release Process
+
+1. Clean up bugs:
+    - Make sure all bugs assigned to the milestone are closed.
+    - Make sure all fixed bugs are assigned to the milestone ([this query][1]
+      should not return anything).
+1. Create a "Version x.y.z." commit ([example][2]) and push it to a branch on your fork.
+1. Ping people on the commit for review.
+1. Testing (post results as comments to commit):
+    - Nightly
+    - Weekly
+    - [Manual testing][3]
+1. If all tests pass and you have LGTM, push to master and tag.
```

Comment:
In fact, we push to scala-js master before running the Nightly and Weekly. Otherwise we can't launch the appropriate Jenkins builds. We tag after, though.